NOTICE

```diff
@@ -1,5 +1,10 @@
-Copyright 2018 Metrological
-Licensed under the Apache License, Version 2.0
+This component contains software that is Copyright (c) 2020 Metrological
```

Comment:
Please use 2018 as the year to match other files.

Reply:
Updated Year to 2018 in Notice file